### PR TITLE
PRO-940 Introduction

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

**Comment by abea** [comment]
... Should this section also talk about what widgets and whatnot are?

File: docs/guide/introduction.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+## What is ApostropheCMS?
+
+ApostropheCMS is a modern, open source, content management system. It's Javascript from front to back, with a Vue.js user interface and Node.js on the server. Importantly, it has everything you might want from a modern CMS, including:
+
+- a headless API built-in
+- a content schema API to easily create any kind of content types you need
+- extensibility through a module ecosystem
+- a great UI for creating content, including in-context editing with content widgets
```
Comment (localghost443) [inline docs/guide/introduction.md:10]:
```suggestion
- a great UI for creating content, including in-context editing
```

File: docs/guide/introduction.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- robust localization to support any number of language or locale content variations
+
+For developers, it offers a site building API that gives you the power to fully customize the website experience without needing to be a Node.js expert. For content editors, the _true_ WYSIWYG experience supports improved editorial flow.
+
+In short, it recognizes pain points on both sides of the CMS experience and relieves them. We can start getting to know that system with two foundational concepts: modules and content schemas.
```
Comment (localghost443) [inline docs/guide/introduction.md:15]:
```suggestion
In short, it recognizes pain points on both sides of the CMS experience and relieves them. We can start getting to know that system with two foundational concepts: modules and content schemas.
```
```suggestion
In short, it recognizes pain points on both sides of the CMS experience and relieves them. Apostrophe is built on two foundational concepts: modules and content schemas.
```

**Review by localghost443 — CHANGES_REQUESTED** [review]
Really really good. Two small adjustment requests.

File: docs/guide/introduction.md (hunk 2 above)
Comment (abea) [reply docs/guide/introduction.md:15]: I'll simply delete this sentence.

**Review by localghost443 — APPROVED** [review]

**Pull request merged**